eclass/kernel-2.eclass

```diff
@@ -211,7 +211,7 @@ esac
 # Added by Daniel Ostrow <dostrow@gentoo.org>
 # This is an ugly hack to get around an issue with a 32-bit userland on ppc64.
 # I will remove it when I come up with something more reasonable.
-[[ ${PROFILE_ARCH} == "ppc64" ]] && CHOST="powerpc64-${CHOST#*-}"
+[[ ${PROFILE_ARCH} == ppc64 ]] && CHOST="powerpc64-${CHOST#*-}"
 
```

Comment:
I think ordinarily we tend to prefer `"${VAR}" == "str"` syntax ...

Reply:
In general we tell people to drop quotation inside ``[[ ]]``.